TFT/src/User/Menu/PrintingMenu.c

```diff
@@ -63,7 +63,7 @@ const char *const speedId[2] = {"Speed", "Flow"};
 bool hasFilamentData;
 
 #define TOGGLE_TIME  2000  // 1 seconds is 1000
-#define LAYER_DELTA  0.1   // minimal layer height change to update the layer display (avoid congestion in vase mode)
+#define LAYER_DELTA  0.01   // minimal layer height change to update the layer display (avoid congestion in vase mode)
```

Comment:
Why so small value (0.01)? Such a small value will defeat its purpose to not overdrive the TFT in vase mode.

Reply:
Haha, it's just for testing. Because the whole icon is not redrawn, it seems acceptable to modify the Z layer in real time, but more testing is needed. I will change it back to 0.1 before merging